src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import './App.css';
+import Shop from './views/Shop';
+
+function App() {
+  return (
+    <div className='App'>
+      <main className='App-main'>
+        <Shop />
```

Comment:
This is a bit overly nested. When this renders to the DOM it will look like this: 

```js
<div id="root">
    <div className='App'>
        <main className='App-main'>
            <Shop>
                ...your shop component details...
            </Shop>
        </main>
    </div>
</div>
```

As you can see, the final render is a bit overly nested. The `<div className='App'>` and `main className='App-main'>` are both extra and redundant.